OptimizelySDK/Event/Dispatcher/HttpClientEventDispatcher45.cs

```diff
@@ -31,33 +31,43 @@ public class HttpClientEventDispatcher45 : IEventDispatcher
         const int TIMEOUT_MS = 10000;
```

Comment:
SUGGEST TIMEOUT_MS = 10000 BE REMOVED.  USE MICROSOFT'S DEFAULT Timeout VALUE.

Microsoft's default HttpClient Timeout is 100 seconds.
```
    HttpClient.Timeout Property
    The default value is 100,000 milliseconds (100 seconds).
    https://msdn.microsoft.com/en-us/library/system.net.http.httpclient.timeout(v=vs.110).aspx
```
Our objective-c-sdk OPTLYHTTPRequestManager.m seems to use Apple NSURLSession's default of 60 seconds.
```
    timeoutIntervalForRequest
    https://developer.apple.com/documentation/foundation/nsurlsessionconfiguration/1408259-timeoutintervalforrequest?language=objc
    The default value is 60.
```

I am unaware of any rationale for decreasing the Microsoft default Timeout from 100 seconds to the 10 seconds which is coded and there isn't any code comment that would explain why Optimizely does this if there is a reason.

Reply:
I have updated PR, https://github.com/optimizely/csharp-sdk/pull/36
I don't recall that there was a reason for doing it.